lms/services/canvas_studio.py

```diff
@@ -67,7 +67,7 @@ def post_load(self, data, **_kwargs):
 
 class APICallInfo(TypedDict):
     path: str
-    authUrl: str | None
+    authUrl: NotRequired[str]
```

Comment:
Not required (ie not dict key) vs optional (key can be None)

Reply:
This reminds me to `undefined` vs `null` in Javascript 😂